.travis.yml

```diff
@@ -60,10 +60,14 @@ matrix:
         - ./ci/suppress_output ./ci/travis/install-dependencies.sh
         - export PATH="$HOME/miniconda/bin:$PATH"
         - ./ci/suppress_output ./ci/travis/install-ray.sh
+        - ./ci/suppress_output ./ci/travis/install-cython-examples.sh
+        - eval "$(curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=master bash)"
       script:
-        - ./ci/suppress_output bash src/ray/test/run_core_worker_tests.sh
-        - ./ci/suppress_output bash streaming/src/test/run_streaming_queue_test.sh
-        - ./java/test.sh
+#        - ./ci/suppress_output bash src/ray/test/run_core_worker_tests.sh
+#        - ./ci/suppress_output bash streaming/src/test/run_streaming_queue_test.sh
+#        - ./java/test.sh
```

Comment:
It will not run the java tests in this suite.

Reply:
fixed

Reply:
fixed